content/modules/ROOT/pages/module-01.adoc

```diff
@@ -248,9 +248,12 @@ Once the repository has synced successfully, you can query for the repository co
 hammer repository info --name "Red Hat Enterprise Linux 9 for x86_64 - AppStream RPMs 9" --product "Red Hat Enterprise Linux for x86_64" --organization "Default Organization" --fields "Content counts/packages"
 ----
 
-Once you have confirmed the package count for the "Red Hat Enterprise Linux 9 for x86_64 - AppStream RPMs 9" repository matches that of the package count from the curl command performed on the CDN, it's time to update the content view associated with the client.
+Once you have confirmed that package count for the "Red Hat Enterprise Linux 9 for x86_64 - AppStream RPMs 9" repository matches that of the package count from the curl command performed on the CDN, it's time to update the content view associated with the client.
 
-Before publishing the content view, it is good practice to check the content view for any filtering that may have been applied to the content view previously and adjust the filters as needed to ensure packages are included/excluded as expected. For this example, there are no content view filters implemented so the package count on the client using the content view should be identical to that of the Satellite. Use the command below to check the content view filters for the `RHEL9-CV` content view:
+Before publishing the content view, it is a good practice to check the content view for any filtering that may have been applied to the content view previously and adjust the filters as needed to ensure that the packages are included/excluded as expected. For this example, there are no content view filters implemented so the package count on the client using the content view should be identical to that of the Satellite. 
```

Comment:
"Before publishing the content view, it is good practice to check the content view for any filtering that may have been applied to the content view previously and adjust the filters as needed to ensure that the packages are included/excluded as expected. For this example, no content view filters are implemented, so the package count on the client using the content view should be identical to that of the Satellite."